src/sliding-sync-sdk.ts

```diff
@@ -463,6 +464,14 @@ export class SlidingSyncSdk {
         // TODO
     }
 
+    /**
+     * Specify the set_presence value to be used for subsequent calls to the Sync API.
+     * @param presence - the presence to specify to set_presence of sync calls
+     */
+    public setPresence(presence?: SetPresence): void {
+        // TODO
```

Comment:
Could do with more information here. Is this impossible with sliding sync, or not yet done in the proxy, or just not yet done in our code?

Reply:
This is just a stub like the 3 methods above it, `syncLeftRooms`, `peek`, `stopPeeking`. Sliding Sync hasn't even began to consider auxiliary features like this.

Reply:
Can you update the TODOs to say that please?